controllers/Category.js

```diff
@@ -0,0 +1,25 @@
+const User = require('../models/User');
+const Package = require('../models/Package');
+const Iternary = require('../models/Iternary');
```

Comment:
Instead of importing from each file like this lets create a `index.js` file in models folder.
Inside that export all these as attributes.
Like in *index.js*
```
module.exports = {
  User: require('./User'),
  Package: require('./Package'),
  // file everything
}
```
And then in this file we all we have to do is require one file.
```
const models = require('../models');
```
Then you can access by models.User etc.

But destructuring provides a better way:
```
const { User, Package } = require('../models');
```
The above code is much simpler.

Reply:
cc @TheMightyRaider

Reply:
Won't the query be like this?


```
const models = require('../models/index');
```

Reply:
Yeah, it can be like that. 
But when you say `require('../somefolder')`, javascript will look into `require('../somefolder/index')` by default. So we don't put that index there.

Reply:
And destructuring makes it easier for you.
1.
```
const { User, Package } = require('../models')
```

2. 
```
const models = require('../models')
const User = models.User
cosnt Package = models.Package
```

1 and 2 are equal. But 1 is more cleaner and shorter.

Reply:
cc: @TheMightyRaider

Reply:
Cool